tutorials/modules.html

```diff
@@ -0,0 +1,646 @@
+    <p>Afterwards you need to use the <code>registerValidator</code> method to register <code>numericValidator</code>:</p>
+    <pre><code class="lang-js">
+// register the validator before using it
+registerValidator(numericValidator);
+    </code></pre>
+    <p>Now, you can use <code>numericValidator</code>, the full example looks like this:</p>
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerValidator, numericValidator } from 'handsontable/validators';
+registerValidator(numericValidator);
+
+// use the numeric validator where you need
+new Handsontable(container, {
+  columns: [
+    {
+      validator: 'numeric',
+    },
+  ]
+// rest of the settings
+});
+            </code></pre>
+            <p>And that is all! You can use the numeric validator!</p>
+        </div>
+        <div class="example-container clearfix">
+          <h3 id="importing-cell-types">Importing cell types</h3>
+              <p>
+                To use a specific cell type you need to import it alongside the registering method. Let's try adding the checkbox cell type.
+              </p>
+              <p>Start with importing the base, <code>CheckboxCellType</code> and the <code>registerCellType</code> method.</p>
+                <pre><code class="lang-js">
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+
+// import the method for registering a cell type
+// import the CheckboxCellType
+import { registerCellType, CheckboxCellType } from 'handsontable/cellTypes';
+    </code></pre>
+    <p>Afterwards you need to use the <code>registerCellType</code> method to register <code>CheckboxCellType</code>:</p>
+    <pre><code class="lang-js">
+// register the cell type before using it
+registerCellType(CheckboxCellType);
+    </code></pre>
+    <p>Now, you can use <code>CheckboxCellType</code>, the full example looks like this:</p>
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerCellType, CheckboxCellType } from 'handsontable/cellTypes';
+registerCellType(CheckboxCellType);
+
+// use the checkbox cell type where you need
+new Handsontable(container, {
+  columns: [
+    {
+      type: 'checkbox',
+    },
+  ]
+// rest of the settings
+});
+                </code></pre>
+                <p>And that is all! You can use the checkbox cell type!</p>
+        </div>
+        <div class="example-container clearfix">
+          <h3 id="importing-locales">Importing locales</h3>
+                  <p>
+                    Importing locales works slightly different than in case of other elements. Let's try adding the PL-pl locale.
+                  </p>
+                  <p>Start with importing the base and the language code:</p>
+                    <pre><code class="lang-js">
+// remember to have the base imported
+import Handsontable from 'handsontable/base';
+
+// import the language code
+import {plPL} from 'handsontable/i18n/languages';
```

Comment:
Most of the imports have space between curly brackets

```suggestion
import { plPL } from 'handsontable/i18n/languages';
```